src/utils/lockfile-manager.ts

```diff
@@ -0,0 +1,71 @@
+// Imports
+import tmp from 'tmp';
+import fs from 'fs';
+import path from 'path';
+
+/** Helper functions to preserve a lockfile */
```

Comment:
This comment is in the wrong place it seems. Also, the functions inside the object need TS Doc (`/** */`) comments too.